src/main/java/uk/ashleybye/httpserver/http/StatusCode.java

```diff
@@ -37,6 +37,18 @@ public String getStatusMessage() {
     }
   },
 
+  MOVED_PERMANENTLY {
```

Comment:
This structure will certainly work to organize status codes and messages, but it does seem a bit verbose. Is there a more concise way to store this information?